### DISCUSSION TITLE
Netlify build error with Tinacms

### DISCUSSION THREAD

**Discussion by growdigital** [tinacms/tinacms#4414] [Help] [answered]
I have TinaCMS working fine locally but when I add `tinacms build &&…` to my `package.json`, Netlify barfs with the error message:

> Build failed due to a user error: Build script returned non-zero exit code: 2

The build command in `netlify.toml` is `yarn project-setup && yarn build`, and [this is the repo](https://github.com/growdigital/hopegarden.uk) 

Thank you!

**Comment by kldavis4** [comment]
Can you provide the full log output?

**Reply by growdigital** [reply, marked as answer]
`9:21:32 PM: Error: Client not configured properly. Missing clientId, token. Please visit https://tina.io/docs/tina-cloud/connecting-site/ for more information`

Yes, I really should read *all* the error messages, not just the bit at the end! :facepalm: Will try now, thank you

**Reply by growdigital** [reply]
Added the environment variables, works a treat, thank you!
